CleverRaven/Cataclysm-DDA
# Modernize integral volume weight

#### Summary

SUMMARY: none

#### Purpose of change

Make JSON values more readable and comprehensible.
Contributes to #36110

#### Describe the solution

Modernizes remaining integral_volume and integral_weight.

#### Describe alternatives you've considered

none

#### Testing

1. Start a new game
2. Make sure it starts successfully

#### Additional context

none
